packages/nimble-components/src/table/styles.ts

```diff
@@ -51,10 +52,11 @@ export const styles = css`
     }
 
     .header-row {
-        display: flex;
-        flex-direction: row;
+        display: grid;
         background: ${applicationBackgroundColor};
         position: relative;
+        width: fit-content;
+        min-width: 100%;
```

Comment:
I can't make a comment on line 69 below, but can we remove the `box-sizing: border-box;` style for `.row` now that the row's height is being set to `controlHeight + 2*borderWidth`?

Reply:
@mollykreis, turns out we still need the `border-box` style for the height to be correct.